src/components/SiteList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+
+import SiteItem from './SiteItem';
+
+class SiteList extends Component {
+  state = {
+    sites: []
+  };
+
+  componentDidMount() {
+    summary.get('/sites').then(response => {
+      this.setState({ sites: response.data.features });
+    });
+  }
+
+  siteSelectHandler(id) {
+    this.props.history.push({ pathname: '/' + id });
+  }
+
+  render() {
+    let sites = this.state.sites.map(site => {
```

Comment:
A couple things here in this block. 

First, you're totally free to keep using `let`, but I want to let you know that `const` will work as well, in the same way putting a const in front of an array or object, then modifying the contents works.

Reply:
yup changed to const!!! previously doing a tutorial, there is problem using const so I switch to let. Not sure what the problem is, but I let you know when I re-encounter it